src/datastructures/Classes.java

```diff
@@ -0,0 +1,151 @@
+package datastructures;
+
+import java.util.ArrayList;
+
+public class Classes {
+	static ArrayList<Entity> entities;
+	static ArrayList<Relationship> relationships;
+
+	//*********************************************************//
+	// Constructor //
+	//*********************************************************//
+
+	public Classes() {
+		entities = new ArrayList<Entity>();
+		relationships = new ArrayList<Relationship>();
+	}
+	
+	//*********************************************************//
+	// Accessors //
+	//*********************************************************//
+	
+	public ArrayList<Entity> getEntities()
+	{
+		return entities;
+	}
+
+	public ArrayList<Relationship> getRelationships()
+	{
+		return relationships;
+	}
```

Comment:
I don't think we need mutators for Classes.java, but I'm curious to hear other opinions on this. Seems like all of the interactions someone might need would be already implemented in other methods.

Reply:
Can't think of a reason why we need mutators atm. All the "setting" is done using methods like "renameClass" and "renameAttribute." So, in a way we already have mutators.

Reply:
Gonna second Stefan's opinion. The only mutating that could or should be done is via the rename functions, which we already have.

Reply:
We probably should make them private if we're going to have a class invariant of "no duplicate names" etc, just to make sure that nothing fishy goes on and causes bugs.